frontend/appflowy_flutter/lib/workspace/application/settings/shortcuts/settings_shortcuts_service.dart

```diff
@@ -0,0 +1,71 @@
+import 'dart:convert';
+import 'dart:io';
+
+import 'package:appflowy/startup/tasks/prelude.dart';
+import 'package:appflowy/workspace/application/settings/shortcuts/shortcuts_model.dart';
+import 'package:appflowy_editor/appflowy_editor.dart';
+
+class SettingsShortcutService {
+  late File file;
+  bool shouldWaitForInitializing = true;
```

Comment:
```suggestion
  final _initCompleter = Completer<void>();
```

Reply:
got to learn something new today, thanks @Xazin!